src/dataviews/dataview-model-base.js

```diff
@@ -263,9 +240,8 @@ module.exports = Model.extend({
   },
 
   update: function (attrs) {
-    if (_.has(attrs, 'source')) {
-      var message = '[SOURCE] Source present in UPDATE attrs. ' + JSON.stringify(attrs.source);
-      track(new Error(message));
+    if (attrs.source) {
```

Comment:
I'd stick to `_.has` because even in the case of someone updating `source` to `undefined` we should throw an error.